xocto/storage/storage.py

```diff
@@ -1179,6 +1184,11 @@ def get_size_in_bytes(self, *, s3_object: S3Object) -> int:
         file_stats = os.stat(filepath)
         return file_stats.st_size
 
+    def get_last_modified(self, *, s3_object: S3Object) -> datetime:
+        filepath = os.path.join(self.storage_root, s3_object.bucket_name, s3_object.key)
+        file_stats = os.stat(filepath)
+        return datetime.datetime(file_stats.st_mtime)
```

Comment:
I think it should be something like this `return datetime.datetime.fromtimestamp(file_stats.st_mtime)`